docs/components/dialog.md

```diff
@@ -238,7 +238,7 @@ Property | Attribute | Type | Default | Description
 
 Method | Parameters | Returns | Description
 --- | --- | --- | ---
-`show` | _None_ | `Promise<void>` | Opens the dialog and fires a cancelable `open` event. After a dialog's animation, an `opened` event is fired.<br>Add an `autocomplete` attribute to a child of the dialog that should receive focus after opening.
+`show` | _None_ | `Promise<void>` | Opens the dialog and fires a cancelable `open` event. After a dialog's animation, an `opened` event is fired.<br>Add an `autofocus` attribute to a child of the dialog that should receive focus after opening.
```

Comment:
Can you fix [this comment in code](https://github.com/material-components/material-web/blob/ffc08d19088ca7b244c55aefbfb3aae92eac8e0c/dialog/internal/dialog.ts#L163) as well? This table is auto-generated.